numpy/ma/core.py

```diff
@@ -4176,13 +4180,23 @@ def count(self, axis=None):
         axis : int, optional
             Axis along which to count the non-masked elements. If `axis` is
             `None`, all non-masked elements are counted.
+        axis : None or int or tuple of ints, optional
```

Comment:
Shouldn't the docstring mention that any further arguments are passed on to `ma.data.count` and `ma.mask.count`?